### ISSUE TITLE
Test audio sample clipped.

### ISSUE THREAD

**Issue by kchanqvq** [zhr1201/deep-clustering#3]
I'm using WSJ0 dataset, and the model convergences. When I use the utility audio_test.py, however, I get only 3 second output when I feed it a 5 second mixed audio. Is that any issue in this python script causing the audio to be clipped?

**Comment by zhr1201** [comment]
Yeah, the test script just uses 3 chunks of frames. You can change that by modifying the AudioSampleReader.gen() into a python generator and iterate through all the frames in audio_test.py.

**Comment by dearwind153** [comment]
@zhr1201 
**Have you ever encountered audio files that cannot be played after voice separation？
I find the separate wav file's format as follow:**
Input File     : 'out_1.wav'
Channels       : 1
Sample Rate    : **16000**
Precision      : **53-bit**
Duration       : 00:00:01.61 = 25792 samples ~ 120.9 CDDA sectors
File Size      : 206k
Bit Rate       : 1.02M
Sample Encoding: **64-bit Floating Point PCM**

**when convert the former wav format as follow, which has't voice.**
**my model is: 
链接:https://pan.baidu.com/s/1jWo9fgoaTTN7Akeh5RbupA  密码:ax1s**

**Can you give me some advice?**

Channels       : 1
Sample Rate    : 8000
Precision      : **16-bit**
Duration       : 00:00:03.13 = 25055 samples ~ 234.891 CDDA sectors
File Size      : 50.2k
Bit Rate       : 128k
Sample Encoding: **16-bit Signed Integer PCM**

**Comment by zhr1201** [comment]
@dearwind153 
All the sample point are all 0 in your file. Guess some error happened and I am not very sure. You probably should run that step by step to see what is wrong.

**Comment by dearwind153** [comment]
@zhr1201 , Ok, thank you for your reply!
